core/domain/suggestion_registry.py

```diff
@@ -283,11 +283,6 @@ def validate(self):
                     suggestion_models.SCORE_TYPE_CONTENT,
                     self.get_score_type()))
 
-        if self.get_score_sub_type() not in constants.ALL_CATEGORIES:
```

Comment:
Make sure that we have a test to validate the pattern of the score_category!

Reply:
We do...